docs/source/user/file_formats.md

```diff
@@ -69,13 +74,15 @@ JupyterLab supports image data in cell output and as files in the above formats.
 * File extension: `.csv`
 * MIME type: None
 
-Files with rows of comma-separated values (CSV files) are a common
-format for tabular data. The default viewer for CSV files in JupyterLab is a high-performance data grid viewer:
+Files with rows of comma-separated values (CSV files) are a common format for
+tabular data. The default viewer for CSV files in JupyterLab is a
+high-performance data grid viewer (which can also handle tab- and
```

Comment:
Is the - after tab a typo?

Reply:
It was intentional, since it's saying `tab-separated and semicolon-separated`.

Reply:
(see https://english.stackexchange.com/a/113433, for example)